Cubical/Data/FinSet/Cardinality.agda

```diff
@@ -0,0 +1,620 @@
+{-
+
+This file contains properties and formulae about cardinality.
+
+-}
+{-# OPTIONS --safe #-}
+
+module Cubical.Data.FinSet.Cardinality where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.HLevels
+open import Cubical.Foundations.Function
+open import Cubical.Foundations.Isomorphism
+open import Cubical.Foundations.Univalence
+open import Cubical.Foundations.Equiv
+open import Cubical.Foundations.Equiv.Properties
+
+open import Cubical.HITs.PropositionalTruncation hiding (elim')
+open import Cubical.HITs.SetTruncation renaming (rec to SetRec ; elim to SetElim) hiding (rec2)
```

Comment:
I suggest using `open import Cubical.HITs.SetTruncation as Set` and then you can write `Set.rec` and `Set.elim`, same for `PropositionalTruncation` and in other files.

Reply:
done